examples/datasets/graph-feature-preprocessor/aml_custom_train.txt

```diff
@@ -1,9 +1,13 @@
 # transaction ID, source account ID, target account ID, timestamp
 0 0 1 0
+0 0 1 0
+0 0 1 0
 1 1 2 1
 2 2 3 2
 3 1 3 3
 4 3 1 4
+4 3 1 4
+4 3 1 4
```

Comment:
FYI, this example works only with the new version of the graph preprocessor that I recently pushed (https://github.ibm.com/ai-foundation/snapml/pull/637). The previous version of the code would report an error due to duplicate edge IDs.

Reply:
yes, thanks. I'm using the main version and it works.